bin/reth/src/node/mod.rs

```diff
@@ -160,19 +170,102 @@ impl Command {
             });
 
         if let Some(tip) = self.tip {
-            debug!("Tip manually set: {}", tip);
+            debug!(target: "reth::cli", %tip, "Tip manually set");
             consensus.notify_fork_choice_state(ForkchoiceState {
                 head_block_hash: tip,
                 safe_block_hash: tip,
                 finalized_block_hash: tip,
             })?;
+        } else {
+            warn!(target: "reth::cli", "No tip specified. reth cannot communicate with consensus clients, so a tip must manually be provided for the online stages with --debug.tip <HASH>.");
         }
 
         // Run pipeline
-        info!("Starting pipeline");
+        info!(target: "reth::cli", "Starting sync pipeline");
         pipeline.run(db.clone()).await?;
 
-        info!("Finishing up");
+        info!(target: "reth::cli", "Finishing up");
         Ok(())
     }
 }
+
+/// The current high-level state of the node.
+#[derive(Default)]
```

Comment:
Not sure about this solution. I don't want us to put everything in the crates on `debug` or lower since crate-consumers will care more about the specific messages than we do for the CLI. An alternative is a custom tracing filter that filters on a field that we can set in the crates, e.g. `notable = true` or something